server/app/api/v1/endpoints/subscriptions.py

```
@@ -0,0 +1,2078 @@
+            "billing_level",
+            "rooftop_id",
+            "rooftop_count",
+            "price_per_rooftop",
+            "total_monthly_price",
+            "metadata",
+        }
+        for field, value in update_data.items():
+            if field not in ALLOWED_UPDATE_FIELDS:
+                continue
+            if field == "metadata":
+                setattr(subscription, "subscription_metadata", value)
```

Ruff (0.14.14)

[warning] Do not call `setattr` with a constant attribute value. It is not any safer than normal property access.

Replace `setattr` with assignment

(B010)